integration/index.ts

```diff
@@ -12,12 +12,14 @@ interface NetlifyCMSOptions {
    * Path at which the Netlify CMS admin dashboard should be served.
    * @default '/admin'
    */
+  disableIdentityWidgetInjection?: boolean;
   adminPath?: string;
   config: Omit<CmsConfig, 'load_config_file' | 'local_backend'>;
   previewStyles?: PreviewStyle[];
 }
 
 export default function NetlifyCMS({
+  disableIdentityWidgetInjection = false;
```

Comment:
This needs to be a comma to separate options here:

```suggestion
  disableIdentityWidgetInjection = false,
```